### ISSUE TITLE
Temporal pole mapping not showing

### ISSUE THREAD

**Issue by LilianYou** [ggseg/ggseg#102]
`
> library(ggseg)
> 
> packageVersion("ggseg")
[1] ‘1.6.5’
> 
> cor_colors <- c("temporal pole"  =   "#FC8D62")
>   
> 
> ggseg(atlas = "dk", mapping = aes(fill = region), colour="gray", size=.5) +
scale_fill_manual(values = cor_colors) +
theme_void()

`
<img width="633" alt="Screenshot 2024-06-04 at 11 29 29 AM" src="https://github.com/ggseg/ggseg/assets/30990195/bba25d9e-5afe-4c63-aa11-da7a842b8c20">

**Comment by booksbythedozen** [comment]
I also recently noted this issue, and the same issue with the frontal pole.

**Comment by drmowinckels** [comment]
Hi.

Thanks for reporting in. This is a known issue, not just with this atlas, but several.
For regions that fall in areas that are not visible from vieweing the lateral or medial sides directly (like the temporal and frontal poles), they wont show in ggseg. This is an issue with smaller regions that fall within "blind" zones.

We have been toying with the idea of slightly shifting the medial view slightly upward and outward, which would show more inferior and caudal areas, and rotating the lateral view slightly down and inward, which would show superior and posterior regions better. However, this would still create issues where other regions dont show easily. 

All in all, there are downsides to plotting 3d surfaces on a 2d plane, and this is one of them.

**Comment by booksbythedozen** [comment]
Thank you so much! This is very helpful. 

I've been using ggseg for clinical purposes and it has proven to be a very powerful tool to facilitate understanding of brain mapping study results among our team. Your comment has helped me better understand the limitations of plotting data this way and relieved any misgivings I had when I first noted this unexpected quirk. 

Perhaps it is time I explore 3D plotting....

Again, many thanks for the quick reply!

**Comment by drmowinckels** [comment]
@booksbythedozen have a look at ggseg3d, which may help. Though downside with that one is that its hard to print images from it.

https://ggseg.github.io/ggseg3d/articles/ggseg3d.html

**Comment by booksbythedozen** [comment]
Yes, I've played with ggseg3d a bit. It's wonderful for illustrating data during case conferences but not ideal as you say for incorporating into printable report documents. Thanks!
